### PR TITLE
Air Quality monitoring device for a small town

### PR DESCRIPTION
**Pull request by LimesKey**
## Describe your project in 1-3 sentences

It's an AirGradient Air Quality sensor for the outdoors. It can test a variety of air quality metrics, mostly notably CO2 which is one of the most looked-at metrics for air quality. I know high CO2 levels are higher in cities and urban neighbourhoods and that CO2 is very bad for the human body; it causes fatigue, anxiety and confusion for most people. I'm excited to build it because I'll get real, hands on experience with soldering and electrical circuitry.

## Hack Club Slack handle
Should be "TastyPastey"

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Review by cfanoulis — CHANGES_REQUESTED** [review]
Hey, are you certain you'll be able to have the parts before Feb. 10th? If not, I'm afraid I cannot approve this grant as we'll be working on projects during 14-23rd of Feb.

File: limeskey.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+For clarification all the parts I'm purchasing are apart of a [guide](https://www.airgradient.com/open-airgradient/instructions/diy-pro/) that has full instructions on assembly. Jeff Geerling did a [video](https://www.youtube.com/watch?v=Cmr5VNALRAg&t=495s) on the project and that is what inspired me. 
+
+## Budget
+
+ For both the PCB and the outdoor enclosure, I already own so I gave a rough estimate on how much it cost. I don't think there is any sales tax for the ones on AliExpress. All prices are in CAD. Note that AliExpress takes a few months to ship to Canada so it'll be a while before I get the parts.
```
Comment (cfanoulis) [inline limeskey.md:20]: Hey! If you already own the items, just add them with a price of 0$

also, do you already have the necessary tools for soldering? if not, you can add them to your budget

**Review by cfanoulis — CHANGES_REQUESTED** [review]
HEY! I see this is urgent, so lets get it done QUICK

One little thing and I'll have you on track for that grant

File: limeskey.md (hunk 1 above)
Comment (LimesKey) [reply limeskey.md:20]: I do have the tools for soldering, recently got them for Christmas!

**Review by cfanoulis — APPROVED** [review]
Great! Let's get your grant setup. A banker from @hackclub/bank-ops will be in touch with more details :)

**Review by daisybanks — APPROVED** [review]
Hey @LimesKey - this looks great! Dming you now on slack to talk next steps in getting you your grant. :)

**Pull request merged**